<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QWidget" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">TEMPO</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="intervalLabel">
     <property name="text">
      <string>Interval</string>
     </property>
     <property name="buddy">
      <cstring>interval</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="windLabel">
     <property name="text">
      <string>Wind</string>
     </property>
     <property name="buddy">
      <cstring>wind</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="gustLabel">
     <property name="text">
      <string>Gust</string>
     </property>
     <property name="buddy">
      <cstring>gust</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="visLabel">
     <property name="text">
      <string>Visibility</string>
     </property>
     <property name="buddy">
      <cstring>vis</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="weatherWithIntensityLabel">
     <property name="text">
      <string>Weather1</string>
     </property>
     <property name="buddy">
      <cstring>weatherWithIntensity</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="weatherLabel">
     <property name="text">
      <string>Weather2</string>
     </property>
     <property name="buddy">
      <cstring>weather</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="cloud1Label">
     <property name="text">
      <string>Cloud1</string>
     </property>
     <property name="buddy">
      <cstring>cloud1</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="cloud2Label">
     <property name="text">
      <string>Cloud2</string>
     </property>
     <property name="buddy">
      <cstring>cloud2</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="cloud3Label">
     <property name="text">
      <string>Cloud3</string>
     </property>
     <property name="buddy">
      <cstring>cloud3</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="cbLabel">
     <property name="text">
      <string>Cumulonimbus</string>
     </property>
     <property name="buddy">
      <cstring>cb</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="name">
     <property name="minimumSize">
      <size>
       <width>76</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(120, 120, 120);</string>
     </property>
     <property name="text">
      <string notr="true">TEMPO</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="interval"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="wind"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="gust"/>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="vis"/>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="weatherWithIntensity"/>
   </item>
   <item row="1" column="6">
    <widget class="QComboBox" name="weather"/>
   </item>
   <item row="1" column="7">
    <widget class="QLineEdit" name="cloud1"/>
   </item>
   <item row="1" column="8">
    <widget class="QLineEdit" name="cloud2"/>
   </item>
   <item row="1" column="9">
    <widget class="QLineEdit" name="cloud3"/>
   </item>
   <item row="1" column="10">
    <widget class="QLineEdit" name="cb"/>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="prob30">
     <property name="text">
      <string notr="true">PROB30</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="prob40">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string notr="true">PROB40</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>interval</tabstop>
  <tabstop>wind</tabstop>
  <tabstop>gust</tabstop>
  <tabstop>vis</tabstop>
  <tabstop>weatherWithIntensity</tabstop>
  <tabstop>weather</tabstop>
  <tabstop>cloud1</tabstop>
  <tabstop>cloud2</tabstop>
  <tabstop>cloud3</tabstop>
  <tabstop>cb</tabstop>
  <tabstop>prob30</tabstop>
  <tabstop>prob40</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
